tachyon/base/files/memory_mapped_file_posix.cc

```diff
@@ -0,0 +1,147 @@
+// Copyright 2013 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "tachyon/base/files/memory_mapped_file.h"
+
+#include <fcntl.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include "tachyon/base/files/file_util.h"
+#include "tachyon/base/logging.h"
+#include "tachyon/base/numerics/safe_conversions.h"
+// #include "tachyon/base/threading/scoped_blocking_call.h"
```

Comment:
Can we remove this line?

Reply:
I'll remain this line to minimize diff and remind me later of uncommenting after we take `ScopedBlockingCall`.